_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {hasError: false};
```

Comment:
I didn't include validations for this component because it is not a required field, so it didn't seem relevant. Is that okay?